client/src/features/ui/modal/Modal.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useRef } from 'react';
+import { useOnClickOutside } from 'usehooks-ts';
+
+
+interface ModalProps {
+  heading: string;
+  setShowModal: any;
+  children: any;
+}
+export const ModalDialog = ({
+    heading,
+    setShowModal,
+    children}: ModalProps) => {
+
+      const ref = useRef(null);
+      useOnClickOutside(ref, () => {
+        setShowModal(false)
+      });
+
+      return (
+        <dialog aria-label={'Modal Dialog'} className="modal modal-open w-full h-full">
+            <div className="modal-box">
+                <h3 className="font-bold text-lg text-primary">{heading}</h3>
+                <button
+                    aria-label={'Close Button'}
```

Comment:
I need to go back and rename some aria labels, but they don't need the 'Button or Dialog' in the name, would you be able to change it for the ones in this file?
https://www.aditus.io/aria/aria-label/